modules/geochemistry/paper.md

```diff
@@ -140,15 +139,47 @@ It is easy to use multiple sub-steps within (1) and/or (2), which may be necessa
 
 ![CPU time required to solve a reactive-transport simulation using the PorousFlow and geochemistry modules.\label{fig:scaling_eg}](joss_paper_scaling.png){ width=60% }
 
-# Example: cooling with feldspars
+# Building and using the code
+
+Building the geochemistry module is a two-step process:
+
+1. Download and install the entire MOOSE package.  Detailed instructions are at the [MOOSE website](https://mooseframework.inl.gov/getting_started/installation/index.html).  Even if only in geochemistry is of interest (without transport, solid mechanics, etc) MOOSE comes as a complete package, so needs to be installed in its entirety.  Depending on the computer setup, this can be straightfoward (on a personal Mac computer) or complicated (on an administered supercomputer).
+
+2. During the installation of MOOSE in part 1, only the "framework" will have been compiled.  To compile all the physics modules, including the geochemistry module, use the following instructions run from the command line:
+```
+cd ~/projects/moose/modules
+make
+cd ~/projects/moose/modules/geochemistry
+make
+```
+(If your computer has $N$ cores, the `make` process may be sped up by using the command `make -j`$N$ instead of simply `make`.)
+
+Check that the geochemistry module is correctly compiled using the following instructions:
+```
+cd ~/projects/moose/modules/geochemistry
+./run_tests
+```
+Virtually all the tests should run and pass.  Some may be "skipped" due to a particular computer setup (for instance, not enough threads).
+
+The geochemistry executable is called `geochemistry-opt` and is found at `~/projects/moose/modules/geochemistry`.  This may be used to run pure geochemistry simulations.  For coupled reactive-transport simulations using the PorousFlow module, the `combined-opt` executable must be used.  For example, to run the Weber-Tensleep GeoTES example from the command line:
+```
+cd ~/projects/moose/modules/combined/examples/geochem-porous_flow/geotes_weber_tensleep
+../../../combined-opt -i exchanger.i
+```
+
+# Tests and examples
+
+The geochemistry module's [code coverage](https://mooseframework.inl.gov/docs/coverage/geochemistry/index.html) currently exceeds 96%.  Many of the more complicated benchmark tests against the Geochemist's Workbench code and the examples are [comprehensively documented](https://mooseframework.inl.gov/modules/geochemistry/tests_and_examples/index.html).  The sections below present one benchmark study and one example by way of illustration.
```

Comment:
96% :O

Reply:
Yea, we have rather strict (unofficial) requirements